contribs/gmf/src/query/windowComponent.js

```diff
@@ -136,13 +136,12 @@ module.component('gmfDisplayquerywindow', queryWindowComponent);
  * @param {!import("ngeo/map/FeatureOverlayMgr.js").FeatureOverlayMgr} ngeoFeatureOverlayMgr The ngeo feature
  *     overlay manager service.
  * @constructor
- * @private
  * @hidden
  * @ngInject
  * @ngdoc controller
  * @ngname GmfDisplayquerywindowController
  */
-function Controller($element, $scope, ngeoQueryResult, ngeoMapQuerent, ngeoFeatureOverlayMgr) {
+export function Controller($element, $scope, ngeoQueryResult, ngeoMapQuerent, ngeoFeatureOverlayMgr) {
```

Comment:
Should be named `QueryWindowComponent`...

Reply:
What about QueryWindowController ^^?

Reply:
:-)

Reply:
Done in this PR: https://github.com/camptocamp/ngeo/pull/4775